src/Queries/Where/Where.ts

```diff
@@ -232,15 +232,18 @@ export class Where {
             operator: 'eq',
           });
         } else if (value !== null && value !== undefined) {
-          for (const operator of operators) {
+          if (typeof value !== 'object') continue;
```

Comment:
Would it work the same to merge this with the `else if`?
So, it's
```ts
} else if (value !== null && value !== undefined && typeof value === 'object');
```

Reply:
Sure, you're right, i'll change it